assets/common/components/register_model/spec.yaml

```diff
@@ -29,12 +29,12 @@ inputs:
   model_name:
     type: string
     optional: true
-    description: Model name to use in the registration. If name already exists, the version will be auto incremented
+    description: Model name to use in the registration. If name already exists, the version will be auto incremented. Model name should satisfy pattern `^[a-zA-Z0-9-]+$`
 
```

Comment:
Would a user friendly explanation make more sense here? instead of adding a regex

Reply:
Good point. Since this was a basic regex I updated it without much explanation. But let me update

Reply:
updated